### PR TITLE
[Merged by Bors] - feat(Data/Matrix/ColumnRowPartitioned): add two negation lemmas

### PR DESCRIPTION
**Pull request by chancenahuway**
add two negation lemmas which state that negating a matrix partitioned by rows/columns is equivalent to negating each row/column of the matrix

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Matrix/ColumnRowPartitioned.lean [hunk 1]
```diff
@@ -140,6 +140,26 @@ lemma transpose_fromRows (A₁ : Matrix m₁ n R) (A₂ : Matrix m₂ n R) :
...
+section Neg
+
+variable [Neg R]
+
+/-- Negating a matrix partitioned by rows is equivalent to negating each of the rows. -/
+@[simp]
+lemma fromRows_neg (A₁ : Matrix m₁ n R) (A₂ : Matrix m₂ n R) :
+    -fromRows A₁ A₂ = fromRows (-A₁) (-A₂) := by
+  ext i j
+  cases i <;> simp [fromRows]
```
Comment (Rida-Hamadani) [inline Mathlib/Data/Matrix/ColumnRowPartitioned.lean:152]:
```suggestion
  ext (i | i) j <;> simp
```

**Comment by chancenahuway** [comment]
I updated the code by adding the two lines suggested by Rida Hamadani.

**Comment by grunweg** [comment]
I presume you're looking for review on this PR, right? Let me label it `awaiting-review` accordingly.
awaiting-review

**Comment by robertylewis** [comment]
Thanks!

bors merge

**Pull request closed without merging**